src/coreclr/inc/readytorun.h

```diff
@@ -404,11 +405,8 @@ enum ReadyToRunHelper
     READYTORUN_HELPER_DblRem                    = 0xE0,
     READYTORUN_HELPER_FltRem                    = 0xE1,
 
-    // These two helpers can be removed once MINIMUM_READYTORUN_MAJOR_VERSION is 10+
-    // alongside the CORINFO_HELP_FLTROUND/CORINFO_HELP_DBLROUND
-    // counterparts and all related code.
-    READYTORUN_HELPER_DblRound                  = 0xE2,
-    READYTORUN_HELPER_FltRound                  = 0xE3,
+    // Formerly READYTORUN_HELPER_DblRound      = 0xE2,
+    // Formerly READYTORUN_HELPER_FltRound      = 0xE3,
```

Comment:
There are some other changes that will be needed for Version 10 (see #98871), as such I think that cleaning up some stuff here isn't a great idea.

Reply:
Good point. I'm breaking the updates out into a separate PR which will need to go in before this one.